local_settings.xml

```diff
@@ -0,0 +1,35 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd">
+    <profiles>
+        <profile>
+            <id>local</id>
+            <repositories>
+                <repository>
+                    <id>openhab-releases</id>
+                    <name>Reposilite Repository</name>
```

Comment:
No longer using reposilite, using gitea but planning to use github for public record of IoT work.